k8s/data-mgt/stage-data-mgt-api.yaml

```diff
@@ -19,6 +19,23 @@ spec:
       labels:
         app: airqo-stage-data-mgt-api
     spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 10
```

Comment:
Preference for deployment goes to the general-purpose nodes takes precedence (with a weight of 10), followed by high-memory nodes (with a weight of 1), and then any other nodes